pkg/commands/service/list.go

```diff
@@ -40,15 +44,23 @@ func (c *ListCommand) Exec(in io.Reader, out io.Writer) error {
 		return fsterr.ErrInvalidVerboseJSONCombo
 	}
 
-	services, err := c.Globals.Client.ListServices(&c.Input)
-	if err != nil {
-		c.Globals.ErrLog.Add(err)
-		return err
+	paginator := c.Globals.Client.NewListServicesPaginator(&c.input)
```

Comment:
Here is an example integration of the go-fastly pagination interface.

It's for the `service list` command but `acl-entry list` and `dictionary-item list` have the exact same logic flow.